source/win-preferences/preferences.vue

```diff
@@ -267,6 +267,17 @@ export default {
         this.populateDynamicValues()
       }
     })
+
+    if (process.env.ZETTLR_IS_TRAY_SUPPORTED === '0') {
+      const leaveAppRunningField = modelToField('system.leaveAppRunning', SCHEMA['tab-advanced'])
+      if (leaveAppRunningField !== undefined) {
+        global.config.set('system.leaveAppRunning', false)
+        leaveAppRunningField.disabled = true
+        if (process.env.ZETTLR_TRAY_ERROR !== undefined) {
+          leaveAppRunningField.info = '⚠️ ' + process.env.ZETTLR_TRAY_ERROR
```

Comment:
I wouldn't use Unicode symbols but rather a Clarity icon for that. That however would need to be implemented in the Vue component. But anyway, I think we can simply remove that icon, since the info string should be explaining quite a lot, shouldn't it?